docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -15,20 +15,20 @@ services apply the change right after custom resource (CR) is patched. Compute d
 
```

Comment:
The introduction to this procedure needs to be rewritten. I need to better understand the purpose of this procedure. 

1. Is this procedure optional or required? 
2. Can this upgrade be performed at any time or does it have to be done immediately after adopting the Compute services to the data plane? Or just when a new version is available? 
3. What is the customer accomplishing in this procedure? All Compute nodes are up to date?

Reply:
after extraction of the FFU guide, we should explicitly state that the fast forward upgrade steps must be completed before starting the verification process

Reply:
@bogdando Can you please review my rewrite to the introduction of the "Performing a fast-forward upgrade of Compute services" procedure? Let me know if I should add anything or edit the text.

Reply:
@bogdando The introduction in your version is the original version that was in this PR.
Original:
"Compute services rolling upgrade cannot be done during adoption,
there is in a lock-step with Compute control plane services, because those are managed independently by data plane ansible and Kubernetes Operators.
{compute_service_first_ref} operator and OpenStack Operator ensure upgrading
is done independently of each other, by configuring
`[upgrade_levels]compute=auto` for Compute services. Compute control plane
services apply the change right after custom resource (CR) is patched. Compute data plane services will catch up the same config change with ansible deployment later on."

But I changed it to:
"You must upgrade the Compute services from Red Hat OpenStack Platform 17.1 to Red Hat OpenStack Services on OpenShift (RHOSO) 18.0 on the control plane and data plane. The upgrade procedure involves the following tasks:

- Update the cell1 Compute data plane services version.
- Remove pre-fast-forward upgrade workarounds from the Compute control plane services and Compute data plane services.
- Run Compute database online migrations to update live data."

Are you ok with the rewrite in this PR, or should we discuss this further?

Reply:
I will rebase my work on top of this (already done!) and we can re-iterate